src/app/app.component.ts

```diff
@@ -57,4 +68,25 @@ export class AppComponent {
     console.log(f.value)
     return f.value;
   }
+
+  liveCheck() {
+    this.onlineEvent = fromEvent(window, 'online');
+    this.offlineEvent = fromEvent(window, 'offline');
+
+    this.subscriptions.push(this.onlineEvent.subscribe(e => {
+      this.liveMessage = 'Connection back online';
+      this.liveStatus = 'online';
+      console.log('Online');
+    }));
```

Comment:
It doesn't look like you use the `e` variable? I assume `e` is for `event`? I wonder if it'd be better not to declare a variable if you're not going to use it. Is it possible to do this in typescript?

```suggestion
    this.subscriptions.push(this.onlineEvent.subscribe(() => {
      this.liveMessage = 'Connection back online';
      this.liveStatus = 'online';
      console.log('Online');
    }));
```

Reply:
Yes, that works completely fine, i think i was toying with a couple ways to approach this function, looks like an old attempt left a scar :p